src/pveclib/vec_f128_ppc.h

```diff
@@ -217,7 +217,7 @@ typedef vf128_t __Float128;
  *  Same as __float128 for PPC.  */
 typedef vf128_t __binary128;
 #ifndef __clang__
-// Clang will allow redefining __float128 even is it not enabled
+// Clang will not allow redefining __float128 even is it not enabled
```

Comment:
should this crap be in 'configure' too?  :-)  (It would be nice to avoid all of the `defined(__clang__) && __clang_major__ < 9` tests that appear below... rather, replace them with something a little more meaninigful?